monasca/README.md

```diff
@@ -493,7 +495,7 @@ Parameter | Description | Default
 `thresh.wait.retries` | Number of startup connection attempts to make before giving up | `24`
 `thresh.wait.delay` | Seconds to wait between retries | `5`
 `thresh.wait.timeout` | Attempt connection timeout in seconds | `10`
-`thresh.heap_ratio` | JVM heap ratio of cgroup limits | `.65`
+`thresh.memory_ratio` | JVM jvm ratio of cgroup limits | `.85`
```

Comment:
> JVM jvm

Might be best to reword this anyway, maybe "Ratio of memory to reserve for the JVM"

Reply:
oops!